packages/atlas/src/hooks/useReactionTransactions.ts

```diff
@@ -107,15 +107,14 @@ export const useReactionTransactions = () => {
             parentCommentId || null,
             proxyCallback(updateStatus)
           ),
-        onTxSync: async ({ block }) => {
+        onTxSync: async ({ transactionHash }) => {
           const refetchResult = await refetchComments()
           setCommentInputIsProcessing({ commentInputId, value: false })
 
-          const newCommentsQueryResult = refetchResult[0]
-          // TODO - We probably shouldn't use inBlock here - it's possible that we could create multiple comments in one block
```

Comment:
Let's update and keep this TODO - ideally we get the ID directly from the status query, without need to find comment by `commentcreatedeventcomment`